src/communications/ssh.communication.ts

```diff
@@ -239,6 +239,9 @@ export class SSHCommunication extends Communication<SSHConnectionData> {
     info(): Promise<FwcAgentInfo> {
         throw new Error("Method not implemented.");
     }
+    systemctlManagement(command: string,service:string): Promise<SystemCtlInfo> {
+        throw new Error("Method not implemented.");
+    }
```

Comment:
We must implement this method for the SSH communication.
Its quite simple, look at other similar methods.